galsim/phase_psf.py

```diff
@@ -817,79 +837,58 @@ def makePSF(self, lam, **kwargs):
         @param circular_pupil      Adopt a circular pupil?  [default: True]
         @param obscuration         Linear dimension of central obscuration as fraction of aperture
                                    linear dimension. [0., 1.).  [default: 0.0]
-        @param nstruts             Number of radial support struts to add to the central obscuration.
-                                   [default: 0]
+        @param nstruts             Number of radial support struts to add to the central
+                                   obscuration. [default: 0]
         @param strut_thick         Thickness of support struts as a fraction of aperture diameter.
                                    [default: 0.05]
-        @param strut_angle         Angle made between the vertical and the strut starting closest to it,
-                                   defined to be positive in the counter-clockwise direction; must be an
-                                   Angle instance. [default: 0. * galsim.degrees]
+        @param strut_angle         Angle made between the vertical and the strut starting closest to
+                                   it, defined to be positive in the counter-clockwise direction;
+                                   must be an Angle instance. [default: 0. * galsim.degrees]
         @param oversampling        Optional oversampling factor *in the image plane* for the PSF
                                    eventually constructed using this Aperture.  Setting
                                    `oversampling < 1` will produce aliasing in the PSF (not good).
                                    [default: 1.0]
         @param pad_factor          Additional multiple by which to extend the PSF image to avoid
                                    folding.  [default: 1.0]
-        @param pupil_plane_im      The GalSim.Image, NumPy array, or name of file containing the pupil
-                                   plane image, to be used instead of generating one based on the
-                                   obscuration and strut parameters.  Note that if the image is saved as
-                                   unsigned integers, you will get a warning about conversion to floats,
-                                   which is harmless. [default: None]
-        @param pupil_angle         If `pupil_plane_im` is not None, rotation angle for the pupil plane
-                                   (positive in the counter-clockwise direction).  Must be an Angle
-                                   instance. [default: 0. * galsim.degrees]
+        @param pupil_plane_im      The GalSim.Image, NumPy array, or name of file containing the
+                                   pupil plane image, to be used instead of generating one based on
+                                   the obscuration and strut parameters.  Note that if the image is
+                                   saved as unsigned integers, you will get a warning about
```

Comment:
Looks like this is no longer a problem.  I tried:

```python
import galsim
import numpy as np

psf = galsim.OpticalPSF(lam=500.0, diam=1.0)
img = galsim.Image(psf._psf.aper.illuminated, scale=psf._psf.aper.pupil_plane_scale, dtype=np.uint16)
img.write("illuminated.fits")
aper = galsim.Aperture(lam=500.0, diam=1.0, pupil_plane_im="illuminated.fits")
psf = galsim.OpticalPSF(lam=500.0, diam=1.0, pupil_plane_im="illuminated.fits")
```

Neither of the last 2 lines raised any warnings, so I'm removing this bit from the appropriate docstrings.